app/phone/src/main/java/dev/jdtech/jellyfin/fragments/SeasonFragment.kt

```diff
@@ -12,12 +18,16 @@ import androidx.lifecycle.lifecycleScope
 import androidx.lifecycle.repeatOnLifecycle
 import androidx.navigation.fragment.findNavController
 import androidx.navigation.fragment.navArgs
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
 import dagger.hilt.android.AndroidEntryPoint
+import dev.jdtech.jellyfin.R
```

Comment:
Can use the core resources like below:

```suggestion
import dev.jdtech.jellyfin.R
import dev.jdtech.jellyfin.core.R as CoreR
```